### PR TITLE
Display Google Map in cuisine.js and nearbySearch the restaurant

### PR DESCRIPTION
**Pull request by ThomasJu**
## Purpose of this request
1. Display a google map in cuisine.js and mark your current location
2. Search for nearby restaurants and print them to a list

## What is being changed?
1. Remove unused index.html
2. Update README for instructions on how to create a Google location API key 
3. Add a google-maps-react-hooks to package.json
4. Update index.css for the CSS container of the map being displayed
5. Add GoogleMap Tag in the HTML returned by cuisine.js to display google maps
6. Call GoogleMap nearby search API to get a list of nearby restaurants and display it on html.

## Preemptive steps before testing
1. `npm install @ubilabs/google-maps-react-hooks -D`
2. Follow the README to create your Google map API keys.

## Testing
1. `npm start`
2. navigate to the Chinese cuisine page and see the map is being displayed
3. click the button that said nearby restaurant search then verify a list of restaurants exists.

### CODE REVIEW

**Review by SeokSong — COMMENTED** [review]
Can you pull and add this map container too cusine.css? index.css was replaced in our last merge

**Review by SeokSong — COMMENTED** [review]
Please create a .gitignore and add package-lock.json and package.json

**Comment by SeokSong** [comment]
Idk how git works but you might have to git stash, git pull and git fetch since this was committed after my changes were merged

**Comment by ThomasJu** [comment]
> Please create a .gitignore and add package-lock.json and package.json

I will add .gitignore for package-lock.json. But I don't want to put package.json in .gitignore because I add a API dependency in package.json

File: my-app/src/cuisine.js [hunk 1]
```diff
@@ -11,66 +11,139 @@ function Cuisine() {
...
+                restaurantHtml.appendChild(restaurantInfoDiv);
+                restaurantList.append(restaurantHtml)
+              }
+            }
+        });
+    }
+
+    // TODO add markers of the nearbyRestaurant to the app
+    const addMarker = () => {
+        // console.log(document.getElementById('map'));
```
Comment (kyuhyunp) [inline my-app/src/cuisine.js:116]: Can you remove these?

**Review by kyuhyunp — APPROVED** [review]

**Review by SeokSong — APPROVED** [review]

**Pull request merged**